src/pages/HomePage.tsx

```diff
@@ -11,43 +11,52 @@ interface ICryptoCoin {
   current_price: string
 }
 export default function HomePage() {
-  const [data, setData] = useState<ICryptoCoin[]>([])
   const URL: string = `https://api.coingecko.com/api/v3/coins/markets?vs_currency=eur&order=market_cap_desc&per_page=100&page=1&sparkline=false`
 
-  useEffect(() => {
-    fetch(URL)
-      .then(date => date.json())
-      .then(setData)
-      .catch(err => {
-        console.error(err)
-      })
-  }, [URL])
+  const fetchCoins = async () => {
+    const response = await fetch(URL)
```

Comment:
Is this the only fetch currently in the application? I thought there were more (coin details page?).